invenio_rdm_records/ext.py

```diff
@@ -127,6 +129,13 @@ def init_config(self, app):
         if not app.config.get("COMMUNITIES_NAMESPACES"):
             app.config["COMMUNITIES_NAMESPACES"] = app.config["RDM_NAMESPACES"]
 
+        if not app.config.get("RDM_FILES_DEFAULT_QUOTA_SIZE"):
```

Comment:
Should check for both and also mention in the warning the old variable, so that it is easier to know what to change.